frontend/src/pages/ActivityReport/Pages/__tests__/PrintSummary.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import PrintSummary from '../PrintSummary';
+
+describe('PrintSummary', () => {
+  it('when a reportCreator with name and role is provided, it renders that (without displaying it)', () => {
+    const reportCreator = { name: 'Lois Lane', role: 'Reporter' };
+
+    render(
+      <PrintSummary reportCreator={reportCreator} />,
+    );
+
+    const expectedResult = 'Lois Lane, Reporter';
+    const result = screen.getByText(expectedResult);
+
+    expect(result).not.toBeNull();
+    expect(screen.queryByText('Report Creator')).not.toBeNull();
```

Comment:
How does this test verify that the value is rendered but not displayed?

Reply:
Ah, sorry, yes the test is that things render, not whether it is visible, since the test renderer doesn't apply CSS styles.